tracetool/README.md

```diff
@@ -0,0 +1,157 @@
+# Linux trace tool
+
+
+## Introduction
+
+This is python tool helps to analysis media stack trace logs combining ftrace events from libva, media driver and Linux kernel mode driver (e.g. i915).
```

Comment:
Fix wording:
"This python tool helps to analyze media stack trace logs combining ftrace events from libva, media driver and Linux kernel mode driver (e.g. i915)."

Reply:
fix, thanks for catch